docs/gatsby-config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  siteMetadata: {
+    title: 'Primer CSS',
+    shortName: 'CSS',
+    description: "CSS for GitHub's Primer design system",
+    imageUrl: 'https://user-images.githubusercontent.com/586552/47590375-121cad80-d93a-11e8-89f2-a1cf108e0548.png'
+  },
+  pathPrefix: '/css',
+  plugins: [
+    {
+      resolve: '@primer/gatsby-theme-doctocat',
+      options: {
+        repoRootPath: '..'
+      }
+    },
+    'gatsby-plugin-sass',
+    {
+      resolve: 'gatsby-plugin-svgr',
+      options: {
+        svgo: false
+      }
+    },
+    {
+      resolve: 'gatsby-plugin-google-analytics',
+      options: {
+        trackingId: 'UA-126681523-2',
```

Comment:
Gave you the wrong code

```suggestion
        trackingId: 'UA-126681523-1',
```